index.bs

```diff
@@ -340,6 +341,10 @@ document on
 the attacker should not be able to control any document on
 `example.com` not in the `foo` namespace.
 
+Issue: TODO(devd): Should we also assert that attacker on main/parent origin
```

Comment:
Since I see the relationship as symmetric, I think we should. Feel free to add that in now, or leave this issue in for later.